README.md

```diff
@@ -46,6 +46,8 @@ The project can be built by running a single script from the `misc/build` direct
 
 ### MacOS
 
+Prerequisites: Homebrew.
```

Comment:
```suggestion
Prerequisites: SDL2, spdlog, catch2

Install with brew: `brew install sdl2 spdlog catch2`
Install with macports: 
```

Can you please fill the command for macports?
I'll remove the brew command from the install script afterwards.